python/grass/jupyter/utils.py

```diff
@@ -87,6 +87,156 @@ def reproject_region(region, from_proj, to_proj):
+    """Queries Raster
+
+    :param coord: coordinates given as tuple
+    :param list raster_list: list of raster names
+
+    :return str: formatted output of raster query results
+    """
+    output_list = [
+        """<table border='1'
+        style='border-collapse:collapse;
+        width: 100%;
+        font-size: 12px;'>"""
+    ]
+
+    for raster in raster_list:
+        raster_output = gs.raster.raster_what(map=raster, coord=coord)
+
+        output = f"""<tr style='background-color: #7CFC00;'>
+        <th colspan='3'>Raster: {raster}</th></tr>"""
+        if raster in raster_output[0] and "value" in raster_output[0][raster]:
+            value = raster_output[0][raster]["value"]
+            output += f"""
+            <tr style='background-color: #f0f0f0;'>
+            <td>1</td>
+            <td>Value</td>
+            <td>{value}</td>
+            </tr>
+            """
+
+        output_list.append(output)
+
+    if len(output_list) == 1:
+        return ""
+
+    output_list.append("</table>")
+    output_list.append("<br>")
+    final_output = "".join(output_list)
+    return final_output
+
+
+def query_vector(coord, vector_list, distance):
+    """Queries Vector
+
+    :param coord: coordinates given as tuple
+    :param list vector_list: list of vector names
+
+    :return str: formatted output of vector query results
+    """
+    output_list = [
+        """<table border='1'
+        style='border-collapse:collapse;
+        width: 100%;
+        font-size: 12px;'>"""
+    ]
+
+    for vector in vector_list:
+        vector_output = gs.vector.vector_what(
+            map=vector, coord=coord, distance=distance
+        )
+
+        if len(vector_output[0]) > 2:
+            output_list.append(
+                f"""
+                               <tr style='background-color: #7CFC00;'>
+                               <th colspan='3'>
+                               Vector: {vector}
+                               </th>
+                               </tr>
+                               """
+            )
+
+            items = list(vector_output[0].items())
+
+            for i, (key, value) in enumerate(items[1:], 1):
+                output = ""
+
+                if isinstance(value, dict):
+                    nested_table = """
+                    <table border='1'
+                    style='border-collapse:collapse;
+                    width: 100%;
+                    font-size: 12px;'>
+                    """
+                    for sub_key, sub_value in value.items():
+                        nested_table += f"""
+                        <tr style='background-color: #f0f0f0;'>
+                        <td>{sub_key}</td>
+                        <td>{sub_value}</td>
+                        </tr>
+                        """
+                    nested_table += "</table>"
+                    output += f"""
+                    <tr style='background-color: #f0f0f0;'>
+                    <td>{i}</td>
```

Comment:
Not sure what is this `i` column there for, just remove it.

Reply:
For adding serial no.